.github/workflows/test-account-settings-prs.yml

```diff
@@ -0,0 +1,41 @@
+# Description: This workflow runs unit + e2e tests on PRs
+#   targeting `account-settings/main`.
+#
+# Triggered by:
+#   (1) maintainer pushes a commit to PRs targeting those branches.
```

Comment:
Note, this workflow will not against forks. But since I'll be the primary engineer on this feature branch and I don't use forks, decided to go with this easier option 🤷